### PR TITLE
`automation`:add new workaround for automation python3package lro

### PR DESCRIPTION
**Pull request by wuxu92**
add a workaround for https://github.com/Azure/azure-rest-api-specs/pull/25435

### CODE REVIEW

File: tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_automation_25435.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+func (workaroundAutomation25434) IsApplicable(apiDefinition *models.AzureApiDefinition) bool {
+	return apiDefinition.ServiceName == "Automation" && apiDefinition.ApiVersion == "2022-08-08"
+}
+
+func (workaroundAutomation25434) Name() string {
+	return "Automation / 25434"
+}
+
+func (workaroundAutomation25434) Process(apiDefinition models.AzureApiDefinition) (*models.AzureApiDefinition, error) {
```
Comment (stephybun) [inline tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_automation_25435.go:22]: 25434?

File: tools/importer-rest-api-specs/components/parser/dataworkarounds/workarounds.go [hunk 2]
```diff
@@ -11,6 +11,7 @@ var workarounds = []workaround{
 	workaroundAuthorization25080{},
 	workaroundDigitalTwins25120{},
 	workaroundAutomation25108{},
+	workaroundAutomation25434{},
```
Comment (stephybun) [inline tools/importer-rest-api-specs/components/parser/dataworkarounds/workarounds.go:14]: 25434?

**Review by stephybun — CHANGES_REQUESTED** [review]

File: tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_automation_25435.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
+package dataworkarounds
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/pandora/tools/importer-rest-api-specs/models"
+)
+
+var _ workaround = workaroundAutomation25434{}
+
```
Comment (stephybun) [inline tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_automation_25435.go:10]: Please add a link to the issue here as well as a brief summary on the issue.

**Review by stephybun — CHANGES_REQUESTED** [review]

File: tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_automation_25435.go (hunk 1 above)
Comment (wuxu92) [reply tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_automation_25435.go]: sorry, typo fixed.

File: tools/importer-rest-api-specs/components/parser/dataworkarounds/workarounds.go (hunk 2 above)
Comment (wuxu92) [reply tools/importer-rest-api-specs/components/parser/dataworkarounds/workarounds.go]: updated

File: tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_automation_25435.go (hunk 3 above)
Comment (wuxu92) [reply tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_automation_25435.go:10]: added.

**Review by stephybun — APPROVED** [review]
Thanks @wuxu92 LGTM 🐞

**Pull request merged**